### DISCUSSION TITLE
Blueprint belongs to / hasMany

### DISCUSSION THREAD

**Discussion by HazemMohamed252** [statamic/cms#5733] [Q&A] [answered]
I want to create new blueprint and add field refer to belongs to that collect data from a table into the database but the issue that the belongs to allow one select but i want multi select, how to define multi select please

**Comment by ryanmitchell** [comment, marked as answer]
I'm assuming you're using an add-on like Runway for this? If so, the question is best asked over on the Runway repository.

**Reply by HazemMohamed252** [reply]
Thank you
